src/configure.rs

```diff
@@ -453,6 +453,12 @@ async fn configure(ctx: &Context, param: &mut LoginParam) -> Result<()> {
     drop(imap);
 
     progress!(ctx, 910);
+
+    if ctx.get_primary_self_addr().await.ok().as_deref() != Some(&param.addr) {
```

Comment:
`save_as_configured_params` below uses `addr_cmp` for address comparison, so maybe it is also better to use that here.

Reply:
> maybe one resync too much is better than one missing, so the existing code is probably fine as well.

that was my thought